docs/deployment.md

```diff
@@ -106,6 +112,18 @@ where `{datacenter}` is the name used in the Cassandra configuration / network t
 The script also allows overriding TTL, keyspace name, replication factor, etc.
 Run the script without arguments to see the full list of recognized parameters.
 
+#### TLS support
+
+Jaeger supports TLS client to node connections as long as you've configured
+your Cassandra cluster correctly. After verifying with e.g. `cqlsh`, you can
+configure the collector and query like so:
+
+```
+collector-linux  --cassandra.servers=cassandra --cassandra.tls \
```

Comment:
I would prefer not to direct people to command line arguments, but instead use env variables. So this could be the standard Docker command for the collector with env vars defined. Also, one param per line to make it more readable.
```
docker run \
  -e SPAN_STORAGE_TYPE=cassandra \
  -e CASSANDRA_SERVERS=<...> \
  -e CASSANDRA_TLS=true \
  . . . etc . . .
  jaegertracing/jaeger-collector
```

Reply:
ok, theres some automatic conversion to env variables from the command line args right?

Reply:
yes, `s/[.-]/_/g + toUpperCase`